### ISSUE TITLE
FR: Generate custom cheat sheets

### ISSUE THREAD

**Issue by myfairsyer** [microsoft/vscode#79503]
TL;DR: Consider allowing to locally generate keyboard cheat sheet PDFs with the currently used shortcuts from within vscode.

It's nice to have beautifully designed cheat sheets for the keaboard shortcuts (#14071).

Unfortunately they neither take into account localization nor customization.
So while for the former it would be great if you provided PDFs for the default keymaps for the most prevalent keyboard layouts on your website (not just en-us), the more general solution this ticket is about is to allow vscode to _locally_ create PDFs with the current keyboard settings, if that's not too much of a overhead technically.

While this might open the gates for further requests like picking other commands' shortcuts to be displayed on the cheat sheet or reordering or regrouping the commands or redesigning the sheet itself, I assume it's a perfectly reasonable middle ground to enable users to just generate the very same cheat sheet with the very same design, set of commands, grouping, ordering etc, but taking into account the shortcuts actually set by the user.

**Comment by myfairsyer** [comment]
@chrmarti Can you give some short feedback (1 sentence) about how you feel about this? Thanks a ton!

**Comment by chrmarti** [comment]
The request makes sense. I don't think we have any of the existing cheat sheets automated, so having a dynamically generated cheat sheet is not within easy reach. (/cc @chrisdias)
